deltachat-ios/Controller/Settings/SettingsViewController.swift

```diff
@@ -242,7 +242,7 @@ internal final class SettingsViewController: UITableViewController {
         } else {
             NotificationManager.removeAllNotifications()
         }
-        UserDefaults.standard.synchronize()
```

Comment:
we can probably remove that safely as it was not called always anyways?

documentation says it is "deprecated" only, so not sure what it does in detail or why it was used in the past :)

Reply:
It doesn't do anything, documentation reads

> this method is unnecessary and shouldn't be used.